code/modules/urist/structures/crates.dm

```diff
@@ -67,7 +67,7 @@ All crates that cannot be ordered go here. Please keep it tidy, by which I mean
 		if(trap)
 			if(istype(trap, /obj/structure/mine))
 				var/obj/structure/mine/M = trap
-				M.explode2()
+				M.activate()
```

Comment:
This is intended to be a frag mine, so the path should be changed accordingly

Reply:
mine.activate() is part of the base landmine API, the frag mines now just do a full override of it because Baymines explode stronk.

If we keep it this way, we would be able to create different types of mine traps (e.g. an EMP mine trap) easily via polymorphism.